apps/api/src/app/controllers/command/__snapshots__/command-payload-schemas.spec.ts.snap

```diff
@@ -15,6 +15,93 @@ Object {
 }
 `;
 
+exports[`command payload schemas The schema for command CREATE_BOOK_BIBLIOGRAPHIC_REFERENCE should have the expected value 1`] = `
+Object {
+  "abstract": Object {
+    "coscradDataType": "NON_EMPTY_STRING",
+    "isArray": false,
+    "isOptional": true,
+  },
+  "creators": Object {
+    "coscradDataType": Object {
+      "name": Object {
```

Comment:
Totally random, but we might want the nested schema definitions to have `coscradDataType: NESTED`
so we can do explicit union discrimination on the front-end.